README.md

```diff
@@ -1,6 +1,11 @@
 # rdf-tests
-Repository for the RDF Tests Community Group
+Repository for the [RDF Tests Community Group]
 
 To view (HTML) files can be [read directly](https://w3c.github.io/rdf-tests/).
 
 The index.html files are built using a Ruby tool-chain based on the corresponding manifest.ttl files. Update using `rake`. Requires Ruby, rdf-turtle and json-ld gems to be installed.
+
+NB: as of October 2025, the test suites for RDF and SPARQL are jointly managed the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].
```

Comment:
It was from before October 2025 and the active part of the WG runs until April 2027 (unless ...).

Change to "currently".

The situation for external contributors is potentially different (which sign-up agreement?). 
e.g. How decisions get made to accept PRs from external contributors.

```suggestion
Currently, the test suites for RDF ([rdf/](rdf/)) and SPARQL ([sparql/](sparql/)) are jointly managed by the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].
```